### PR TITLE
PLDM: Fix for OpPanel enable function 34

### PR DESCRIPTION
**Pull request by sagisin**
This commit fixes the incorrect handling of multiple
PDR records modified events sent by the host which are
all sent within fraction of seconds.

Defect: SW546471

Signed-off-by: Sagar Srinivas <sagar.srinivas@ibm.com>

### CODE REVIEW

File: host-bmc/host_pdr_handler.cpp [hunk 1]
```diff
@@ -806,13 +806,27 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
                                     }
                                 }
                             }
+                            modifiedCounter--;
                         }
                     }
-                    else
+                    else if ((isHostPdrModified != true) &&
```
Comment (Pavithrab7) [inline host-bmc/host_pdr_handler.cpp:812]: Can you add a comment here why this change was made and if there is any optimal solution other than this

**Review by Pavithrab7 — APPROVED** [review]

File: host-bmc/host_pdr_handler.cpp [hunk 2]
```diff
@@ -761,7 +761,7 @@ void HostPDRHandler::processHostPDRs(mctp_eid_t /*eid*/,
                 }
                 else
                 {
-                    if (isHostPdrModified)
+                    if ((isHostPdrModified == true) || !(modifiedCounter == 0))
```
Comment (manojkiraneda) [inline host-bmc/host_pdr_handler.cpp:764]: isHostPdrModified is a bool , we dont need to compare that to true again ?

**Review by manojkiraneda — APPROVED** [review]
